New features

qbit- opened the pull request:
Feature extractor has been rewritten to coincide with nvidia nemo. 
The interface of the following was changed:
FeatureExtractor
utils.read_audio

RunFMe committed:
Add rnnt model with working convertation

RunFMe committed:
Add converter and rnnt

RunFMe committed:
separate example

RunFMe committed:
Update tflite model versions

RunFMe committed:
Add support for libri speech download

RunFMe committed:
Add Mozilla DeepSpeech loading

Commit:
small refactoring and fixes

Commit:
added notebooks folder with experiments

Commit:
New dataset preparation script

Commit:
Added new datasets

RunFMe committed:
Add eager mode for ctc pipeline

RunFMe committed:
Fix predict in ctc pipeline

Commit:
Merge branch 'master' into qbit

RunFMe committed:
Refactor loss

Commit:
prepare_libri corrected

RunFMe committed:
Change deepspeech model to avoid materializing unneccesary tensor

RunFMe committed:
Move create_overlapping_windows to utils

RunFMe committed:
Fix imports and model

RunFMe committed:
Fix typo in load_mozilla_deepspeech

Commit:
Merge branch 'master' into qbit

Conflicts:
	automatic_speech_recognition/model/__init__.py
	automatic_speech_recognition/model/deepspeech.py

Commit:
Merge branch 'master' into qbit

Conflicts:
	automatic_speech_recognition/model/deepspeech.py

Commit:
fixed wrapper

vmarkiNN committed:
Add files via upload

Added quartznet model

Commit:
Fix masking tflite

vmarkiNN committed:
Add files via upload

vmarkiNN committed:
Add files via upload

Commit:
reverted to passing lengths through self

vmarkiNN committed:
inference example, not working

Commit:
clelaning

Commit:
kwargs for compile

Commit:
added mixed precision

Commit:
revert policy added

Commit:
training scripts and pipeline

vmarkiNN committed:
Add files via upload

RunFMe committed:
Remove SetMask layer

Commit:
change training code

Commit:
Resolve memory problem for ctc loss

RunFMe committed:
Merge pull request #1 from qbit-/speedup_training

Speedup training

Commit:
train with horovod

Commit:
Merge branch 'master' into qbit

Commit:
tested pipeline

Commit:
train script update

Commit:
unification of arguments ds2

Commit:
Merge branch 'qbit' of https://github.com/qbit-/Automatic-Speech-Recognition into qbit

Commit:
Merge branch 'Quartznet' into qbit

Commit:
merge quartznet

Commit:
code cleaning

Commit:
working prediction

vmarkiNN committed:
Update quartznet_infer.py

Commit:
updated read_audio and alphabet

Commit:
added blank to alphabet

Commit:
new feature extractor wip

Commit:
feature extractor wip 2

Commit:
feature extractor finished

rolczynski commented:
hey folks @ValeriyMarkin @qbit- @RunFMe !
I am super glad that you use this package. I wonder maybe do you wish to make a valid PR to the master hm?

qbit- commented:
Hi Rafal, thanks for a great code!
We chose your project because it is simple and modular. 
Our stuff is currently very much WIP and poorly tested, and many things are broken. 
We would be glad to PR after the master branch gets cleaner.
By the way, we mostly did the following:
1) implemented a new feature extraction part
2) added a couple of models
3) used horovod parallelization instead of keras MirroredStrategy

rolczynski commented:
Great!
Please do not forget about this PR 👍  
Of course, I can help you somehow.
